Pipfile

```diff
@@ -10,15 +10,15 @@ ufo2ft = "==2.30.0"
 fontmake = "==3.5.*"
 fontbakery = "==0.10.4"
 skia-pathops = "==0.8.*"
-gftools = "==0.9.*"
+gftools = "*"
```

Comment:
Please pin the version to whatever GF requires.
Accepting "any version" can break CI (it has happened in the past)